support/utils_test.go

```diff
@@ -7,19 +7,77 @@ import (
 	"encoding/hex"
 	"errors"
 	"fmt"
+	"github.com/FactomProject/factoid"
 	. "github.com/pegnet/pegnet/support"
 	"testing"
 )
 
-func TestBurn(t *testing.T) {
+func TestConvertFctToPegAssets(t *testing.T) {
```

Comment:
Should probably include hard-coded test vectors to verify addresses do not change in the future.

FCT -> PNT addr list